### ISSUE TITLE
Manually bump OIS in Airnode

### ISSUE THREAD

**Issue by aquarat** [api3dao/airnode#1430]
## Description

Airnode's CI flow currently doesn't use frozen lock files. This means that even patch releases of OIS get installed and this breaks airnode-examples (eg. '1.1.1' won't match the current version of '1.1.2').

To resolve this with minimal fuss for the v0.8.0 Airnode release the ois version was pinned to '1.1.1' (instead of '^1.1.1').

The package will need to be manually updated in future, paying special attention to the airnode-examples.

There may be a slicker alternative to the above, feel free to suggest something or raise it on Slack.

Also refer to https://github.com/api3dao/airnode/pull/1428

**Comment by Siegrift** [comment]
Found the cause of the issue and it's because of how we create our docker containers... Working on a fix.

**Comment by Siegrift** [comment]
TLDR: There is nothing wrong, but we need to use a specific (pinned) version of the OIS package. See: https://github.com/api3dao/ois/issues/31#issuecomment-1241046424

The reason we have this failure on the CI, specifically in airnode-examples is because we use the snapshot packages published to local NPM registry. They are regular NPM packages (e.g. dist folder is published, **without** dependencies). The docker containers then use the local registry to install that package. This installs the published content of the monorepo package, but it also installs all the transitive dependencies (e.g. the content of the package.json) and it adheres to the semver, since there is no yarn.lock anymore. This results in @api3/ois@1.1.2 being installed.

**Comment by Siegrift** [comment]
I've rebased @amarthadan PR to master (with pinned ois version) https://github.com/api3dao/airnode/tree/deployments-common-bucket-emo and the CI is passing. 

Moving the issue to DONE. This is something that needs to be mentioned in OIS for which there is a separate issue: https://github.com/api3dao/ois/issues/31
